generated/nifake/nifake.proto

```diff
@@ -687,11 +715,13 @@ message InitWithOptionsRequest {
   bool id_query = 3;
   bool reset_device = 4;
   string option_string = 5;
+  nidevice_grpc.SessionInitializationBehavior requested_behavior = 6;
```

Comment:
Note: This update to InitWithOptions for the Request and Response is the interesting bit and the new functionality tested in this PR. The other added functions were pulling in recent hapigen updates to change those methods from 'private' to 'public' for nimi-python purposes.